packages/manager/src/features/Billing/PdfGenerator/PdfGenerator.ts

```diff
@@ -58,11 +58,18 @@ const addLeftHeader = (
   const isAkamaiBilling = getShouldUseAkamaiBilling(date);
   const isInternational = !['US', 'CA'].includes(country);
 
-  const remitAddress = isAkamaiBilling
-    ? ['US', 'CA'].includes(country)
-      ? ADDRESSES.akamai.us
-      : ADDRESSES.akamai.international
-    : ADDRESSES.linode;
+  const getRemitAddress = (isAkamaiBilling: boolean) => {
+    // M3-6218: Temporarily change "Remit To" address for US customers back to the Philly address
+    if (!isAkamaiBilling || country === 'US') {
+      return ADDRESSES.linode;
+    }
+    if (['CA'].includes(country)) {
+      return ADDRESSES.akamai.us;
+    }
+    return ADDRESSES.akamai.international;
+  };
```

Comment:
This change is entirely optional and only affects the presence of a single return statement. It does not make any significant difference.
```suggestion
let address = ADDRESSES.akamai.international;
    if (!isAkamaiBilling || country === 'US') {
      address=  ADDRESSES.linode;
    }
    if (['CA'].includes(country)) {
      address = ADDRESSES.akamai.us;
    }
    return address;
  };
```